eo-maven-plugin/src/test/java/integration/MojasIT.java

```diff
@@ -0,0 +1,197 @@
+                pulled
+            ),
+            temp.resolve(pulled).toFile().exists(),
+            Matchers.is(true)
+        );
+    }
+
+    @Test
+    void assemblesNotFailWithFailOnError(@Mktmp final Path temp) throws IOException {
+        final String prog = String.join(
+            "\n",
+            "+alias stdout org.eolang.io.stdout",
+            "+home https://github.com/objectionary/eo",
+            "+package test",
+            "+version 0.0.0",
+            "",
+            "[x] < wrong>",
+            "  (stdout \"Hello!\" x).print"
+        );
+        new Farea(temp).together(
+            f -> {
+                f.clean();
+                f.files()
+                    .file("src/main/eo/one/main.eo")
+                    .write(prog.getBytes(StandardCharsets.UTF_8));
+                f.build()
+                    .plugins()
+                    .appendItself()
+                    .execution("tests")
+                    .goals("register", "parse", "shake");
+                f.exec("test");
+            }
+        );
+        MatcherAssert.assertThat(
+            "Even if the eo program invalid we still have to parse it, but we didn't",
+            temp.resolve(String.format("target/eo/%s", ParseMojo.DIR)).toAbsolutePath(),
+            new ContainsFiles(String.format("**/main.%s", AssembleMojo.XMIR))
+        );
+        MatcherAssert.assertThat(
+            "Even if the eo program invalid we still have to optimize it, but we didn't",
+            temp.resolve(String.format("target/eo/%s", ShakeMojo.DIR)).toAbsolutePath(),
+            new ContainsFiles(String.format("**/main.%s", AssembleMojo.XMIR))
+        );
+    }
+
+    @Test
+    void configuresChildParameters(@Mktmp final Path temp) throws IOException {
+        new Farea(temp).together(
+            f -> {
+                f.clean();
+                f.files()
+                    .file("src/main/eo/one/main.eo")
+                    .write(MojasIT.helloWorld().getBytes(StandardCharsets.UTF_8));
+                f.build()
+                    .plugins()
+                    .appendItself()
+                    .execution("tests")
+                    .goals("register", "assemble")
+                    .configuration()
+                    .set("trackTransformationSteps", Boolean.TRUE.toString());
+                f.exec("test");
+            }
+        );
+        MatcherAssert.assertThat(
+            "AssembleMojo should have configured parameters within the Mojos that it uses, but it didn't",
+            temp.resolve(
+                String.format("target/eo/%s/one/main.%s", ShakeMojo.DIR, AssembleMojo.XMIR)
+            ).toFile().exists(),
+            Matchers.is(true)
+        );
+    }
+
+    private static String helloWorld() {
+        return String.join(
+            "\n",
+            "+alias stdout org.eolang.io.stdout",
+            "+home https://www.eolang.org",
+            "+package foo.x",
+            "+unlint object-has-data",
```

Comment:
@Suban05 try to remove as many `+unlint` metas as you can. They should have been fixed already